knowledge/guidelines.rst

```diff
@@ -43,6 +43,8 @@ Good practices
   the graph to avoid the conflicts without overrides and forces is the recommended approach.
 - **Please, do not abuse 'tool_requires'**. Those are intended only for executables like ``cmake`` and ``ninja`` running in the "build"
   context, not for libraries or library-like dependencies, that must use ``requires`` or ``test_requires``.
+- Positional arguments when calling Conan should be specified first, before any named argument. For example,
+  ``conan install . -s os=Windows`` is correct, but ``conan install -s os=Windows .`` is not.
```

Comment:
```suggestion
  ``conan list "zlib/*" -r=conancenter`` is correct, but ``conan -r=conancenter "zlib/*"`` is not.
```

maybe this is a bit more clear?

Reply:
I like the ``conan install .`` example, why do you think the ``conan list`` is better?

Reply:
The positional argumwent being a point might get missed by readers, whereas the pattern in the list is a bit more prominent, but let's merge as-is if it looks good for you